web/src/features/fba/components/viz/ElevationFlag.tsx

```diff
@@ -1,40 +1,30 @@
 import React from 'react'
-import { Box, Grid, Typography } from '@mui/material'
-
-const FLAG_COLOUR = '#CCCCCC'
+import { Typography } from '@mui/material'
+import Grid from '@mui/material/Unstable_Grid2'
```

Comment:
Why do we need to import `Unstable_Grid2` over `Grid`?

Reply:
`@mui` has refactored the `Grid` component to use Flexbox instead of the css grid. In `@mui` v5 the implementation is available via `Unstable_grid2`. Using The unstable grid 2 import now could make things a little easier when we move to `@mui` v6.